b2sdk/scan/__init__.py

```diff
@@ -0,0 +1,9 @@
+######################################################################
```

Comment:
is empty __init__ needed in python3?

Reply:
Let's try :) I remember having some troubles when making pypi package without `__init__.py`, but maybe it was me who misconfigured something.